docs/whats-news/2022/2022-08.md

```diff
@@ -0,0 +1,39 @@
+---
+id: 2022-08
+title: What's Newsletter August 2022
+slug: /2022-08
+---
+
+# What's New in Nimbus August 2022
+
+## Welcome
+
+The Nimbus team will have a **demo booth at #moz-aloha** All-Hands at the end of September! Come find us, chat about experimentation, and maybe even grab a sticker!
+
+## Exciting Changes
+
+### Mobile First Run
+
+Instrument Nimbus SDK in telemetry in order to provide experimental treatments during the onboarding experience.
+- This was affecting more than 100,000 clients and over 1.5M events/requests.
+
+### Cancel Review
+
+If you launch, end, or end enrollment for an experiment it automatically goes to Review. Now, there is an option
+to cancel that change and go back to an idle state where changes can be made.
+
+### UI/UX Updates
+
+Various user experience updates to streamline workflows and provide more consistency throughout the tool, like:
```

Comment:
Also in terms of UI improvements, you could add `You can see all dates with the year now`- something like that